Formula/p/podlet.rb

```diff
@@ -0,0 +1,27 @@
+class Podlet < Formula
+  desc "Generate podman quadlet files from a podman command or compose file"
+  homepage "https://github.com/containers/podlet"
+  url "https://github.com/containers/podlet/archive/refs/tags/v0.2.4.tar.gz"
+  sha256 "3774e49ee0826221772c06d37478c1cd421d8cbf00cda6b16605df4b9f8b2292"
+  license "MPL-2.0"
+  head "https://github.com/containers/podlet.git", branch: "main"
+  livecheck do
+    url :head
+  end
```

Comment:
```suggestion
```

Reply:
Is the livecheck not needed anymore?

Reply:
The head url is the same as the download url. No need to specify a livecheck with that